### PR TITLE
ignore emacs temporary files

### PR DESCRIPTION
**Pull request by uhoreg**
and fix ignoring of dotfiles

### CODE REVIEW

**Comment by uhoreg** [comment]
@anoadragon453 and/or @turt2live can you take a look at this one?

File: scripts/continuserv/main.go [hunk 1]
```diff
@@ -120,8 +120,11 @@ func filter(e fsnotify.Event) bool {
 		return false
 	}
 
-	// Avoid some temp files that vim writes
-	if strings.HasSuffix(e.Name, "~") || strings.HasSuffix(e.Name, ".swp") || strings.HasPrefix(e.Name, ".") {
+	_, fname := filepath.Split(e.Name)
```
Comment (turt2live) [inline scripts/continuserv/main.go]: This doesn't work correctly on Windows: https://play.golang.org/p/FrVhH1c8cdC

Comment (richvdh) [reply scripts/continuserv/main.go]: it should, if you run it on Windows?

Comment (turt2live) [reply scripts/continuserv/main.go]: +10 idiot points to me

**Comment by turt2live** [comment]
This has been approved for a while - merging. (also it gets a +1 from me)

**Pull request merged**